ci/sonar-scanner.sh

```diff
@@ -66,9 +66,11 @@ pushd "workers/unity"
     # Wildcards don't appear to play nice with this.
     args+=("-d:sonar.cs.opencover.reportsPaths=$(find ./logs -name "*.xml" | paste -sd "," -)")
 
+    dotnet new globaljson --sdk-version 2.2.402 --force
     traceStart "Execute sonar-scanner :sonarqube:"
         dotnet-sonarscanner begin "${args[@]}"
         dotnet msbuild ./unity.sln -t:Rebuild -nr:false | tee "${PROJECT_DIR}/logs/sonar-msbuild.log"
         dotnet-sonarscanner end "-d:sonar.login=${TOKEN}" | tee "${PROJECT_DIR}/logs/sonar-postprocess.log"
     traceEnd
+    rm -f global.json
```

Comment:
This won't happen if something breaks in the scanner. This should be replaced with an exit trap to ensure the global.json is ALWAYS removed

Reply:
That's not a problem since if the scanner breaks, the job ends and the build dir gets wiped (including this file)